docs_src/OVMS/runObjectDetectionPipelineWithNewModel.md

```diff
@@ -0,0 +1,33 @@
+# Run Object Detection Pipeline with New Model
+For running object detection pipeline, we have defaulted the model to person_vehicle_bike_detection_2000. You can use different model to run object detection. Here are the steps:
+
+1. Add new section to config file for model server
+2. Download new model
+3. Add new model label file
+4. Add architecture type of new model as input to pipeline
+5. rebuild containers
+
+
+## Add New Section to Config File for Model Server
+Here is the config file location: `configs/opencv-ovms/models/2022/config.json`, edit the file and append following configuration section template
+```json
+,
+{"config": {
+                "name": "modelName",
+                "base_path": "/models/modelName/FP16-INT8",
+                "nireq": 1,
+                "batch_size":"1",
+		        "shape": "auto",
```

Comment:
We need to say that `shape` will depend on the model. Sometimes "auto" works, but sometimes you need to know the shape value or don't add `shape` attribute at all.